roles/mysql/tasks/main.yml

```diff
@@ -1,17 +1,22 @@
 ---
 # tasks file for mysql
 #
-- name: MySQL | add mariadb repo keys
+- name: MySQL | Install MariaDB key
   ansible.builtin.apt_key:
-    keyserver: "keyserver.ubuntu.com"
-    id: "0xF1656F24C74CD1D8"
+    url: https://mariadb.org/mariadb_release_signing_key.asc
+    state: present
+  register: install_app_key
 
-- name: MySQL | add maria db repository
+- name: MySQL | Install MariaDB repository
   ansible.builtin.apt_repository:
     repo: "deb [arch=amd64] https://mirrors.accretive-networks.net/mariadb/repo/10.2/ubuntu {{ ansible_distribution_release }} main"
+    state: present
+    validate_certs: false
```

Comment:
Can we get cert validation working? Also, should this be a handler?

Reply:
it can't be a handler. We want to get this certificate. Not all repos do TLS. So if we switch to new mirror we don't want to get the... needs TLS cert.